### ISSUE TITLE
License URL in third-party file

### ISSUE THREAD

**Issue by simonklb** [mojohaus/license-maven-plugin#75]
It would be nice to have the possibility to include more than just the license name in the third party file. Any chance that the license URL could be made available to the template file as well?

**Comment by jahlborn** [comment]
Yes, this would be very useful especially for "custom" licenses.  Today, we are just including the url as part of the name, e.g. "project--1.0=Custom License - http://project/license".  Maybe the license plugin could parse that and split it into name and url info?

**Comment by tchemit** [comment]
@simonklb @jahlborn which file are you talking about ?

the missing license file ? (THIRD-PARTY.properties).

Yes it would be a great idea. The best to thing to do would to use here one of the predefined license name (named a gplv3, agpl, ...) this name is related to a full definition of a license .

Do you get me ?

**Comment by simonklb** [comment]
@tchemit Sorry, I'm not really following you here. What we want is to be able to link to the license via an URL. This is mostly needed for custom licenses, like @jahlborn mentioned. For well-known licenses there is no problem for people to find them online.

**Comment by jahlborn** [comment]
To confirm, yes we are referring to the THIRD-PARTY.properties file.  And to reiterate what @simonklb said, this is desirable for custom licenses, e.g. jdom http://www.jdom.org/docs/faq.html#a0030 .

**Comment by jom0000** [comment]
ควย
